src/components/AddBoardModal/AddBoardModal.js

```diff
@@ -11,27 +12,25 @@ class AddBoardModal extends Component {
 
  render() {
    return (
-      <div className="modal-body">
-        <form>
-          <div className="form-group">
+      <ModalBody>
+        <Form>
```

Comment:
Es ist möglich in diesem Formular Enter zu drücken und dann wird das Formular abgeschickt. Du solltest statt onClick auf dem Button lieber onSubmit auf dem Formular verwenden und dann mit event.preventDefault() das abschicken verhindern.